SPECS/rubygem-bundler/rubygem-bundler.spec

```
@@ -3,14 +3,15 @@
 %global gem_name bundler
 Summary:        manages an application's dependencies
 Name:           rubygem-bundler
-Version:        1.16.4
-Release:        5%{?dist}
+Version:        2.3.8
+Release:        1%{?dist}
 License:        MIT
 Vendor:         Microsoft Corporation
 Distribution:   Mariner
 Group:          Development/Languages
-URL:            https://rubygems.org/gems/%{gem_name}/versions/%{version}
-Source0:        https://rubygems.org/downloads/%{gem_name}-%{version}.gem
+URL:            https://github.com/rubygems/rubygems
+#Source0:        https://github.com/rubygems/rubygems/archive/refs/tags/bundler-v%{version}.tar.gz
+Source0:        rubygems-%{gem_name}-v%{version}.tar.gz
```

This is preferred now (makes automation/PR checks easier):

```suggestion
Source0:        https://github.com/rubygems/rubygems/archive/refs/tags/bundler-v%{version}.tar.gz#/rubygems-%{gem_name}-v%{version}.tar.gz
```